ORStools/proc/directions_lines_proc.py

```diff
@@ -102,13 +104,23 @@ def processAlgorithm(self, parameters, context, feedback):
             context
         )
 
+        # parameters[self.IN_FIELD] returns a PyQt5.QtCore.QVariant with "NULL" as content
+        # in case of absence of self.IN_FIELD.
+        # qgis overwrites this type's __bool__ in
+        # https://github.com/qgis/QGIS/blob/master/python/PyQt/PyQt5/QtCore.py:
+        # def __bool__(self):
+        #     return not self.isNull()
+        # The check below works because of that.
         source_field_name = parameters[self.IN_FIELD]
+        if source_field_name:
+            get_field_options = dict(
```

Comment:
`get_field_options` is not initialized like this if `source_field_name` is false

Reply:
done